AdobeStockImage/Model/SaveMediaGalleryAssetInterface.php

```diff
@@ -0,0 +1,28 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\AdobeStockImage\Model;
+
+use Magento\Framework\Api\Search\Document;
+use Magento\Framework\Exception\CouldNotSaveException;
+
+/**
+ * Used for getting a media gallery asset id from document and asset save.
+ */
+interface SaveMediaGalleryAssetInterface
+{
+    /**
+     * Process saving MediaGalleryAsset based on the search document and destination path.
+     *
+     * @param Document $document
+     * @param string $destinationPath
+     *
+     * @return int
+     * @throws CouldNotSaveException
+     */
+    public function execute(Document $document, string $destinationPath): int;
```

Comment:
Why the return type is `int`?

Reply:
We return type id to process the next step of saving the Adobe Stock asset. We need it to save asset keywords and the adobe stock asset itself.

Reply:
I believe we need to return `void`. The same like we did for `SaveImageFileInterface`.

Reply:
@lenaorobei @sivaschenko I am going to provide changes in a hour.

Reply:
@lenaorobei @sivaschenko I provided requested changes. Now `SaveImageFileInterface` also returns `void`. I used directly a command which gets media asset by a path.